df_py/util/blocktime.py

```diff
@@ -116,7 +116,11 @@ def __init__(self, target_timestamp):
             self.target_timestamp = target_timestamp
 
         def timeSinceTimestamp(self, block_i):
-            block_timestamp = web3.eth.get_block(int(block_i)).timestamp
+            try:
+                block_timestamp = web3.eth.get_block(int(block_i)).timestamp
+            except Exception as e:
+                print(f"An exception occurred while getting block {block_i}, {e}")
+                block_timestamp = 0
```

Comment:
So if the exception happens, then `block_timestamp = 0`

And it returns `block_timestamp - self.target_timestamp` = `0 - self.target_timestamp` = `-self.target_timestamp`.

This seems wrong.